src/config/2019.json

```
@@ -2,8 +2,25 @@
   "settings": [
     {
       "is_live": true,
-      "supported_languages": ["en","es","fr","hi","it","ja","nl","pt","ru","tr","uk","zh-CN","zh-TW"],
-      "ebook_languages": ["en","ja"]
+      "supported_languages": [
+        "en",
+        "es",
+        "fr",
+        "hi",
+        "it",
+        "ja",
+        "nl",
+        "pt",
+        "ru",
+        "tr",
+        "uk",
+        "zh-CN",
+        "zh-TW"
+      ],
+      "ebook_languages": [
+        "en",
+        "ja"
+      ]
```

Lets just collapse again, as pretty short:

```suggestion
      "supported_languages": ["en","es","fr","hi","it","ja","nl","pt","ru","tr","uk","zh-CN","zh-TW"],
      "ebook_languages": ["en","ja"]
```

---

wow, it's my "format on paste" IDE setting... Need to switch it off.

---

Your IDE is probably correct in most cases 😁

But in this case I think overkill.